src/main/java/org/jenkinsci/plugins/credentialsbinding/MultiBinding.java

```diff
@@ -80,21 +80,37 @@ public final String getCredentialsId() {
     public static final class MultiEnvironment implements Serializable {
 
         private final Map<String,String> values;
+        private final Map<String,String> secretValues;
```

Comment:
This is going to be null after deserialization following a restart with a running Pipeline build, leading to `NullPointerException`s.